recipes/njoy2016/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set org = "njoy" %}
+{% set name = "NJOY2016" %}
+{% set version = "2016.66" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```

Let's remove the necessary jinja variables and make the URL a bit more readable.

```suggestion
  url: https://github.com/njoy/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```